File: sdk/javascript/examples/quickstart.mjs
// IOTYPE_TOKEN=... node examples/quickstart.mjs
import { Iotype } from "@iotype-ai/sdk";

const io = new Iotype();

console.log("translate:", await io.translate("سلام! امروز هوا بسیار عالی است.", "fa", "en"));
console.log("synthesize:", await io.synthesize("سلام دنیا", { speaker: "tanaz" }));

const files = await io.files();
console.log(`files: ${files.length} submitted`);
for (const f of files.slice(0, 5)) console.log(` ${f.uuid}  ${f.filename}`);

File: sdk/python/examples/quickstart.py
"""Run the synchronous endpoints end to end.

    export IOTYPE_TOKEN="..."
    python examples/quickstart.py
"""
from iotype import Iotype, IotypeError

def main() -> None:
    io = Iotype()

    print("translate:", io.translate("سلام! امروز هوا بسیار عالی است.", "fa", "en"))

    url = io.synthesize("سلام دنیا", speaker="tanaz", tone="general")
    print("synthesize:", url)
    if url:
        io.download(url, "narration.mp3")
        print("saved -> narration.mp3")

    files = io.files()
    print(f"files: {len(files)} submitted")
    for f in files[:5]:
        print(f"  {f.uuid}  {f.filename}  done={f.done}")

if __name__ == "__main__":
    try:
        main()
    except IotypeError as exc:
        raise SystemExit(f"iotype error: {exc}")
